drivers/serial/uart_esp32.c

```diff
@@ -971,8 +970,7 @@ static const DRAM_ATTR struct uart_driver_api uart_esp32_api = {
 #define ESP_UART_DMA_INIT(n)                                                                       \
 	.dma_dev = ESP32_DT_INST_DMA_CTLR(n, tx),                                                  \
 	.tx_dma_channel = ESP32_DT_INST_DMA_CELL(n, tx, channel),                                  \
-	.rx_dma_channel = ESP32_DT_INST_DMA_CELL(n, rx, channel),                                  \
-	.uart_id = (DEVICE_DT_GET(DT_NODELABEL(uart0)) != DEVICE_DT_INST_GET(n)),
+	.rx_dma_channel = ESP32_DT_INST_DMA_CELL(n, rx, channel)
```

Comment:
nit: it's usually a good idea to leave the trailing `,` on the last element so you don't get this odd diffs where nothing really changed